src/openms/source/FEATUREFINDER/FeatureFindingMetabo.cpp

```diff
@@ -307,6 +309,7 @@ namespace OpenMS
     
     isotope_filtering_model_ = param_.getValue("isotope_filtering_model").toString();
     use_smoothed_intensities_ = param_.getValue("use_smoothed_intensities").toBool();
+    report_smoothed_intensities_ = use_smoothed_intensities_ && param_.getValue("report_smoothed_intensities").toBool();
```

Comment:
we could issue an OPENMS_LOG_WARN if "report_smoothed_intensities" is true but "use_smoothed_intensities" is false